qcfractal/interface/models/rest_models.py

```diff
@@ -575,6 +575,25 @@ def ensure_list_of_dict(cls, v):
 
 register_model("result", "GET", ResultGETBody, ResultGETResponse)
 
+### Wavefunction data
+
+
+class WavefunctionStoreGETBody(ProtoModel):
+    class Data(ProtoModel):
+        id: ObjectId = Schema(None, description="Id of the Wavefunction Key/Value Storage object to get.")
+
+    meta: QueryMetaProjection = Schema(QueryMetaProjection(), description=common_docs[QueryMetaProjection])
+    data: Data = Schema(
+        ..., description="Data of the Wavefunction Get field: consists of a ObjectId of the Wavefunction object to fetch.")
+
+
+class WavefunctionStoreGETResponse(ProtoModel):
+    meta: ResponseGETMeta = Schema(..., description=common_docs[ResponseGETMeta])
+    data: Dict[str, Any] = Schema(..., description="The entries of the Wavefunction object requested.")
+
+
+register_model("wavefunctionstore", "GET", WavefunctionStoreGETBody, WavefunctionStoreGETResponse)
```

Comment:
another nitpick: consider naming the endpoint "wavefunction" to match the names of the other models ("molecule", "result", etc.)

Reply:
This seems much more analogous to the KVStore than the `result` or `molecule`. This is why the API and REST endpoints match other stores. What do you think?

Reply:
I didn't fully appreciate that, and I am okay with WavefunctionStore now.